src/renderer/components/image-button/index.test.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import * as renderer from 'react-test-renderer';
+
+import BackButton from '.';
+
+describe('back button', () => {
```

Comment:
Let's use the component name as a convention, i.e. `describe('BackButton', // ...`

Reply:
That said, I don't recommend using `BackButton` to test `ImageButton`, especially you are using snapshot testing. That means, if you ever add functionalities to `BackButton`, it's going to break the test for `ImageButton`. That's unnecessary maintenance work. Just write an `ImageButton` usage in the test, and do snapshot testing by adjusting the props, or interacting with it, etc.

Reply:
I hadn't even looked at this file - it's legacy from copying the back button to use as a base for the image button - I'll go thru and add the proper test for the ImageButtton

Reply:
I've changed the test - does it just run when the code is rendered? or is there a way I can run this test specifically?